instruqt-tracks/sentinel-for-terraform-v4/09-private-module-registry-v2/assignment.md

```diff
@@ -0,0 +1,105 @@
+---
+slug: private-module-registry-v2
+id: vkoczgu1dkmz
```

Comment:
Let's clear the `id:` fields out of `assignment.md` while we're moving to v3 of the track config